middleware/redirects.ts

```diff
@@ -0,0 +1,10 @@
+export default function ({ store, redirect, route }): void {
+  const prefix = store.getters.currentUrlPrefix
+
```

Comment:
missing check for starts with prefix

Reply:
I dont get this one, starts with?

Reply:
@vikiival

Reply:
like every route has name 

`/rmrk/gallery` `rmrk-gallery`
`/bsx/gallery` `bsx-gallery`
`/spotlight` `spotlight`

so if someone adds route 
`/collections` your current impl will redirect him/her to `/${prefix}/explore`